vico/core/src/main/java/com/patrykandpatrick/vico/core/cartesian/CartesianChart.kt

```diff
@@ -77,7 +77,7 @@ public open class CartesianChart(
   @get:RestrictTo(RestrictTo.Scope.LIBRARY_GROUP) public val marker: CartesianMarker? = null,
   protected val markerVisibilityListener: CartesianMarkerVisibilityListener? = null,
   @get:RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-  public val layerPadding: CartesianLayerPadding = CartesianLayerPadding(),
+  public val getLayerPadding: ((ExtraStore) -> CartesianLayerPadding) = { CartesianLayerPadding() },
```

Comment:
For simplicity, we don't use the `get` prefix for `ExtraStore` lambdas. They're essentially just scopes for expressions.

You can find a similar approach in other libraries, e.g. the Kotlin standard library:

```kt
inline fun check(
    value: Boolean, 
    lazyMessage: () -> Any
)
```

```suggestion
  public val layerPadding: ((ExtraStore) -> CartesianLayerPadding) = { CartesianLayerPadding() },
```